packages/transmute-dashboard/src/components/Messages/Messages.js

```diff
@@ -0,0 +1,209 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import AppBar from '../AppBar';
+
+import IPFS from 'ipfs';
+import Room from 'ipfs-pubsub-room';
+import _ from 'lodash';
+
+import { withStyles } from 'material-ui/styles';
+import Card, { CardActions, CardContent, CardHeader } from 'material-ui/Card';
+import Button from 'material-ui/Button';
+import Grid from 'material-ui/Grid';
+import { LinearProgress } from 'material-ui/Progress';
+import Input, { InputLabel } from 'material-ui/Input';
+import { FormControl } from 'material-ui/Form';
+import Select from 'material-ui/Select';
+import List, { ListItem, ListItemText } from 'material-ui/List';
+
+import theme from '../../theme';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  flex: {
+    flex: 1,
+  }
+});
+
+const ipfsOptions = {
+  EXPERIMENTAL: {
+    pubsub: true
+  },
+  config: {
+    Addresses: {
+      Swarm: [
+        "/dns4/ws-star.discovery.libp2p.io/tcp/443/wss/p2p-websocket-star"
```

Comment:
this config object should probably be imported and not embedded in a component.

Reply:
Agree, good catch.